src/Engine/Renderer/Texture/TextureManager.hpp

```diff
@@ -59,6 +59,9 @@ class RA_ENGINE_API TextureManager final
      */
     Texture* getOrLoadTexture( const TextureParameters& texParameters, bool linearize = false );
 
+    // helper function, load the texture without adding it to the manager.
```

Comment:
This is not a valid doxygen documentation comment. Write a full valid documentation for this method.